src/code_80091750.c

```diff
@@ -527,41 +528,31 @@ void func_800921B4(void) {
 GLOBAL_ASM("asm/non_matchings/code_80091750/func_800921B4.s")
 #endif
 
-#ifdef MIPS_TO_C
-//generated by mips_to_c commit 3c3b0cede1a99430bfd3edf8d385802b94f91307
-s32 func_800B4520(); // extern
-? set_text_color(s32); // extern
-extern s32 gGlobalTimer;
+void set_text_color(s32);
+extern s32 func_800B4520();
 
-void func_800921C0(s32 arg0, s32 arg1, s32 arg2) {
-    if (arg0 == arg1) {
+void func_800921C0(s32 test, s32 target, s32 alternative_color) {
+    if (test == target) {
         if (func_800B4520() != 0) {
             set_text_color(gGlobalTimer % 3);
-            return;
         }
-        set_text_color(5);
-        return;
+        else{
```

Comment:
else goes on the same line as the if closing bracket
```
if () {

} else {

}

```

Reply:
Done